settings.py

```diff
@@ -83,3 +84,32 @@
 # https://docs.djangoproject.com/en/1.6/howto/static-files/
 
 STATIC_URL = '/static/'
+
```

Comment:
Why enable tracking in this settings file - in order to test that the tracking took place? Seems like we'd need to add a test for that...